src/seedu/addressbook/data/AddressBook.java

```diff
@@ -74,4 +74,11 @@ public boolean equals(Object other) {
                 || (other instanceof AddressBook // instanceof handles nulls
                         && this.allPersons.equals(((AddressBook) other).allPersons));
     }
+
+    /**
+     * Sorts the array based on the comparator defined in person.
+     */
+    public void sort (){
```

Comment:
inconsistency in space placing - do remember to adhere to our coding standard for java.